enhanced-resolve/index.d.ts

```diff
@@ -0,0 +1,134 @@
+// Type definitions for enhanced-resolve v2.3.0
+// Project: http://github.com/webpack/tapable.git
```

Comment:
This looks like the wrong link. Did you mean https://github.com/webpack/enhanced-resolve?
Would also be nice to have a documentation link in a comment since it's not documented at the repository.